docs/7_updates.md

```diff
@@ -75,17 +80,26 @@ We can push this new application to your development board through Device Manage
 Run:
 
 ```
-$ mbed device-management update device -D YOUR_ENDPOINT_NAME
+$ mbed dm update device -D YOUR_ENDPOINT_NAME
 ```
 
 Replace `YOUR_ENDPOINT_NAME` with the endpoint name in Device Management.
 
-Inspect the logs on the device (via a serial monitor) to see the firmware update progress. It looks similar to:
+Inspect the logs on the device (via a serial monitor) to see the firmware update progress (this may take a few minutes). It looks similar to:
```

Comment:
```suggestion
Inspect the logs on the device (via a serial monitor) to see the firmware update progress. (This may take a few minutes.) It looks similar to:
```